content/docs/trust/trust-manager/README.md

```diff
@@ -131,7 +131,13 @@ JKS has been supported since v0.5.0, and PKCS#12 since v0.7.0.
 
 Applications consuming JKS and PKCS#12 trust stores often require a password to be set for legacy reasons. These passwords are often security theater - either they use very weak encryption or the passwords are provided in plaintext next to the files they encrypt which defeats the purpose of having them.
 
-Trust bundles do not contain private keys, and so for most use cases there wouldn't be any security benefit to encrypting them. As such, passwords for trust stores are hard-coded to `changeit` for JKS and `""` (the empty string or "password-less") for PKCS#12. Future releases of trust-manager may make these passwords configurable.
+Trust bundles do not contain private keys, and so for most use cases there wouldn't be any security benefit to encrypting them. As such, passwords for trust stores are set by default to `changeit` for JKS and `""` (the empty string or "password-less") for PKCS#12. 
+
+Recent releases will allow you to change that password by setting the bundle yaml file `spec.target.additionalFormats.jks.password` and `spec.target.additionalFormats.pkcs12.password`. 
+
+Older releases have the current default values hard-coded and they can not be changed. For more information read [why password are not helpful](../../faq/#why-are-passwords-on-jks-or-pkcs12-files-not-helpful).
```

Comment:
suggestion: I think this link isn't in the correct format - there's some documentation [here](https://github.com/cert-manager/website/?tab=readme-ov-file#task-linking-to-other-documentation-pages) on the format we need.

It's kinda specific and I realise that's annoying - sorry!

Reply:
Sorry, I could've been clearer on my comment! The relevant bit is at the bottom of that link:

> ✨ When linking to an "Introduction" page, link to the README.md file directly. The framework will remove the whole filename, so e.g. `[example link](../usage/README.md)` will link to `https://cert-manager.io/docs/usage/`.

So I think the link should be to `faq/README.md`

Reply:
Oh, also, I added a shorter link for that section since I thought we'd link to it frequently (see [here](https://github.com/cert-manager/website/blob/2191b5f9214c1009cc5cfc886a4703361a04d0d8/content/docs/faq/README.md#L172)): 

`../../faq/README.md#keystore-passwords`

Reply:
done